terraform/modules/stack/base/outputs.tf

```diff
@@ -65,11 +65,20 @@ output "rds_postgres_security_group" {
 
 /* RDS Bosh Instance */
 output "bosh_rds_url" {
```

Comment:
Does it make sense to do the same pattern we've used elsewhere for output and suffix with current/previous?